.github/workflows/ci.yml

```diff
@@ -100,6 +100,10 @@ jobs:
           sudo apt update
           sudo apt install libpulse-dev libegl1-mesa libopengl0 gstreamer1.0-gl
       - uses: tlambert03/setup-qt-libs@v1
+      - name: Install Linux system packages
+        if: contains(matrix.os, 'ubuntu')
+        shell: bash
+        run: sudo apt install libpulse-dev libegl1-mesa libopengl0 gstreamer1.0-gl
```

Comment:
I think this is duplicating line 101. Maybe the step at lines 96 to 101 need to be after the step that uses the `tlambert03/setup-qt-libs` action for things to work? If the step command duplication is necessary, could we add a comment clarifying why this is needed and maybe rename the step so we don't have two steps in the job with the same name?

Reply:
No. This is an artifact from rebasing that I did not notice.